jetty-websocket/javax-websocket-server/src/main/java/org/eclipse/jetty/websocket/javax/server/JavaxWebSocketServerContainer.java

```diff
@@ -199,20 +234,11 @@ public void addEndpoint(Class<?> endpointClass) throws DeploymentException
         else
         {
             if (deferredEndpointClasses == null)
-            {
                 deferredEndpointClasses = new ArrayList<>();
-            }
             deferredEndpointClasses.add(endpointClass);
         }
     }
 
-    /**
-     * Register a ServerEndpointConfig to the server
-     *
-     * @param config the endpoint config to add
-     * @throws DeploymentException if unable to deploy that endpoint class
-     * @see javax.websocket.server.ServerContainer#addEndpoint(ServerEndpointConfig)
-     */
     @Override
```

Comment:
Restore javadoc

Reply:
Don't need javadoc on overrided methods that already have javadoc.